packages/formation/sass/modules/_m-loading-indicator.scss

```diff
@@ -3,7 +3,7 @@
 
   .loading-indicator {
     min-height: 8em;
-    background: url(/img/loading-state.svg) center center no-repeat;
+    background: url(../assets/img/loading-state.svg) center center no-repeat;
```

Comment:
`url-loader` and `svg-url-loader` look for these files relative to the entry file (`./sass/full.scss`) and does nothing when they cannot find them. This updates the references so these loaders copies them into the directories according to the rules in the webpack config.